### ISSUE TITLE
Make it easy to generate PDF for downstream and users

### ISSUE THREAD

**Issue by tbm** [zacchiro/ledger2beancount#169]
This is mostly a question for @jelmer although I'm also looking for input from @zacchiro and @wzyboy.

I was looking at the Debian package of ledger2beancount (thanks @jelmer) and was wondering if it would make sense to ship the manual as a PDF.  It's 15+ pages and some users might prefer to read a PDF or print it.

I've added metadata to the `.md`, so you can just run `pandoc -f markdown manual.md -o manual.pdf`.

The question is: should we do anything else to make this easy for users and downstream distributors?

I don't really want to distribute the PDF itself, but I was thinking we should add a `pdf` target in the `Makefile` to run pandoc (and remove the .pdf in the `clean` target) and maybe add something to the docs about building the PDF (BTW, I'm wondering if we should move all the installation related info from the manual to a separate document; after all, if you're reading the PDF, you probably installed ledger2beancount already).

Any opinions?  And if you agree, can you send a patch for the Makefile.

**Comment by zacchiro** [comment]
Sure, a `make doc` Makefile target would be nice to have. (And FWIW I agree with you we shouldn't distribute the PDF via Git, of course.)

**Comment by wzyboy** [comment]
I was wondering whether distributing PDF manuals is common so I checked:

```
$ pacman -Ql | grep '\.pdf$'
fontconfig /usr/share/doc/fontconfig/fontconfig-devel.pdf
fontconfig /usr/share/doc/fontconfig/fontconfig-user.pdf
ghostscript /usr/share/doc/ghostscript/9.26/GS9_Color_Management.pdf
graphite /usr/share/doc/graphite2/api/refman.pdf
ijs /usr/share/doc/ijs/ijs_spec.pdf
libxml2 /usr/share/doc/libxml2-2.9.9/html/tutorial/xmltutorial.pdf
lirc /usr/share/doc/lirc/lirc.org/PCB.pdf
msmtp /usr/share/doc/msmtp/msmtp.pdf
openexr /usr/share/doc/openexr/InterpretingDeepPixels.pdf
openexr /usr/share/doc/openexr/MultiViewOpenEXR.pdf
openexr /usr/share/doc/openexr/OpenEXRFileLayout.pdf
openexr /usr/share/doc/openexr/ReadingAndWritingImageFiles.pdf
openexr /usr/share/doc/openexr/TechnicalIntroduction.pdf
openexr /usr/share/doc/openexr/TheoryDeepPixels.pdf
speex /usr/share/doc/speex/manual.pdf
speexdsp /usr/share/doc/speexdsp/manual.pdf
t1lib /usr/share/t1lib/doc/t1lib_doc.pdf
```

So it's common :-)

**Comment by tbm** [comment]
Ok, I added a `Makefile`.  I'll leave it up for distributors if they want to ship the PDF.  I also added man pages, btw (generated with `scdoc`).

**Comment by tbm** [comment]
@zacchiro any opinion on:

> BTW, I'm wondering if we should move all the installation related info from the manual to a separate document; after all, if you're reading the PDF, you probably installed ledger2beancount already

**Comment by jelmer** [comment]
It would make sense to ship the manual in the package. Thanks for adding the make bits.

**Comment by tbm** [comment]
Fixed in 56a753b323283eba20bb8a97affc97bd9b381abe
